Applications/Spire/Include/Spire/TimeAndSales/TimeAndSalesTableView.hpp

```diff
@@ -46,6 +49,7 @@ namespace Spire {
       boost::signals2::scoped_connection m_model_end_loading_connection;
       SnapshotLoadingWidget* m_loading_widget;
       TransitionWidget* m_transition_widget;
+      std::unordered_map<int, int> m_minimum_column_widths;
```

Comment:
Use an array and define the columns, or use an existing definition if it's available, ie.

```static constexpr auto PRICE_COLUMN = 0;```